docs/src/config.md

```diff
@@ -20,3 +22,51 @@ catch
     @warn "Could not turn on Rebugger key bindings."
 end
 ```
+
+## Customize keybindings
+
+It's possible that Rebugger's default keybindings don't work for you.
+This can be for various reasons. Some window managers override the Rebugger's
+keybindings with their own. Some terminals map the keybindings used by Rebugger
+to different escape sequences than those hardcoded in Rebugger. You can work
+around these issues by adding your own keybindings.
+
+To add your own keybindings, use `Rebugger.add_keybinding(action, keybinding)`.
+This can be done during a running Rebugger session. Here is an example that
+maps the "step in" action to the key "F5":
+```julia
+julia> Rebugger.add_keybinding(action=:stacktrace, keybinding="\e[15~")
```

Comment:
This seems more cumbersome than what I suggested. It seems like this can be just
```julia
Rebugger.add_keybinding(stacktrace="\e[15~")
```
and write `add_keybinding` as:
```julia
julia> add_keybinding(; override=false, kwargs...) = for (kw,val) in kwargs @show kw val end
add_keybinding (generic function with 1 method)

julia> add_keybinding(stacktrace='a', stepin='b')
kw = :stacktrace
val = 'a'
kw = :stepin
val = 'b'
```
You can of course still check whether `kw in keys(keybindings)`.

Reply:
Ha, yeah, didn't think of it this way.